internal/server/event_handlers.go

```diff
@@ -72,6 +73,17 @@ func (s *EventServer) handleEvent() func(w http.ResponseWriter, r *http.Request)
 				continue
 			}
 
+			// skip alert if the message matches a regex from the exclusion list
+			if len(alert.Spec.ExclusionList) > 0 {
+				for _, exp := range alert.Spec.ExclusionList {
+					if r, err := regexp.Compile(exp); err == nil {
```

Comment:
Not sure if swallowing the error is the right thing to do, as regexp can be tricky to configure.

Reply:
Returning the error doesn't make sense, as the sender will retry loop and cause DDOS.

Reply:
Log it?